### PR TITLE
Fixes #1315, Add documentation for openID in development

### LINKED ISSUE #1315: Document openid implementation better
While there is some mention about implementing openid in our installation instructions, this is mainly intended for production.

It is very unclear to new contributors as to what needs to be done in order to implement openid in development - or that this is even possible!

### PR DESCRIPTION
**Pull request by BelieveC**
Fixes #1315
Add documentation to use openID in development.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ GitHub issues are the primary way for communicating about specific proposed chan
 ### Email Notifications
 **Note**: We use [letter_opener](https://github.com/ryanb/letter_opener) in development environment. You can check out your mails by visiting [localhost:3000/letter_opener](http://localhost:3000/letter_opener).
 
+### openID
+In order to use [openID](http://openid.net/) logins for your OSEM in development you need create `config/secrets.yml` file. After this simply refer to development block in `config/secrets.yml.example` to setup `config/secrets.yml`. Now you can run `rake secret` to generate secret_key. Replace sample `secret_key_base` in `config/secrets.yml` with generated one. Restart rails server once to use openID logins.
```
Comment (ChrisBr) [inline CONTRIBUTING.md]: Why not just copy the example? 
` cp config/secrets.yml.example config/secrets.yml` ?

Comment (ChrisBr) [inline CONTRIBUTING.md]: Also generating a secret is not really necessary, just remove the comment should be enough

Comment (ChrisBr) [inline CONTRIBUTING.md]: @differentreality is that really everything what is necessary to setup openID?

Comment (BelieveC) [inline CONTRIBUTING.md]: Yeah we can `cp config/secrets.yml.example config/secrets.yml `. PR is related to development so I kept it to development setup of openID. Should I change it  `cp config/secrets.yml.example config/secrets.yml ` ?

**Comment by differentreality** [comment]
We are currently using dotenv for configuration, and it is not documented in your changes. Moreover this is not explaining what the user can do with openid in development / how things will work. I will close this, as it needs quite some re-thinking and re-writing, which is why #1315 was created.

**Comment by BelieveC** [comment]
Thanks! I will have deeper analysis next time.

**Pull request closed without merging**